totest-manager.py

```diff
@@ -725,6 +725,25 @@ def get_current_snapshot(self):
         return self.iso_build_version(self.project + ':ToTest', self.main_products[0])
 
 
+class ToTest150Ports(ToTestBaseNew):
+    main_products = [
+        '000product:openSUSE-cd-mini-aarch64',
+        '000product:openSUSE-dvd5-dvd-aarch64',
```

Comment:
Wasn't there ppc64le as well?

Reply:
There is ppc64le as well, but it doesn't build yet. I will add it once it does in a new PR